bazel.spec

```diff
@@ -28,8 +28,12 @@ Patch1: bazel-absl
 
 %build
 
-export EXTRA_BAZEL_ARGS="--host_javabase=@local_jdk//:jdk --jobs %{compiling_processes}"
-export BAZEL_CXXOPTS="-Wno-error=deprecated:-std=c++%{cms_cxx_standard}"
+# Workaround for compilation issues in upb
+%define cms_cxx_standard 17
+
+export EXTRA_BAZEL_ARGS="--define=ABSOLUTE_JAVABASE=${JAVA_ROOT} --jobs %{compiling_processes}"
+export BAZEL_CXXOPTS="-std=c++%{cms_cxx_standard}"
+export JNI_FLAGS="--add-opens=java.base/java.nio=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED"
```

Comment:
may be this should be done for java17 only? i.e. if java version in 17 or above then set this?